Index: cinnamon/files/usr/share/cinnamon/applets/keyboard@cinnamon.org/applet.js
===================================================================
--- cinnamon.orig/files/usr/share/cinnamon/applets/keyboard@cinnamon.org/applet.js	2016-11-19 12:27:17.350569751 +0100
+++ cinnamon/files/usr/share/cinnamon/applets/keyboard@cinnamon.org/applet.js	2016-11-20 11:13:32.250094025 +0100
@@ -213,7 +213,7 @@
             if (this.show_flags) {
                 let name = this._config.get_icon_name_for_group(i);
 
-                let file = Gio.file_new_for_path("/usr/share/iso-flag-png/" + name + ".png");
+                let file = Gio.file_new_for_path("/usr/share/iso-flags-png-320x240/" + name + ".png");
 
                 if (file.query_exists(null)) {
                     actor = new EmblemedIcon(file.get_path(), this._config.get_flag_id_for_group(i), "popup-menu-icon").actor;
@@ -262,7 +262,7 @@
         if (this.show_flags) {
             let name = this._config.get_current_icon_name();
 
-            let file = Gio.file_new_for_path("/usr/share/iso-flag-png/" + name + ".png");
+            let file = Gio.file_new_for_path("/usr/share/iso-flags-png-320x240/" + name + ".png");
 
             if (file.query_exists(null)) {
                 this._applet_icon = new EmblemedIcon(file.get_path(), this._config.get_current_flag_id(), "applet-icon");
