--- a/files/usr/bin/cinnamon-slideshow
+++ b/files/usr/bin/cinnamon-slideshow
@@ -2,5 +2,5 @@
 
 import os
 
-os.execvp("/usr/lib/cinnamon-slideshow/cinnamon-slideshow.py", ("",))
+os.execvp("/usr/share/cinnamon/cinnamon-slideshow/cinnamon-slideshow.py", ("",))
 
--- a/generate_cs_module_desktop_files.py
+++ b/generate_cs_module_desktop_files.py
@@ -22,9 +22,9 @@
 from gi.repository import GLib
 
 try:
-    sys.path.append('files/usr/lib/cinnamon-settings/modules')
-    sys.path.append('files/usr/lib/cinnamon-settings/bin')
-    mod_files = glob.glob('files/usr/lib/cinnamon-settings/modules/*.py')
+    sys.path.append('files/usr/share/cinnamon/cinnamon-settings/modules')
+    sys.path.append('files/usr/share/cinnamon/cinnamon-settings/bin')
+    mod_files = glob.glob('files/usr/share/cinnamon/cinnamon-settings/modules/*.py')
     mod_files.sort()
     if len(mod_files) is 0:
         raise Exception("No settings modules found!!")
