--- a/files/usr/bin/cinnamon-slideshow
+++ b/files/usr/bin/cinnamon-slideshow
@@ -2,4 +2,4 @@
 
 import os
 
-os.system("/usr/lib/cinnamon-slideshow/cinnamon-slideshow.py")
+os.system("/usr/share/cinnamon/cinnamon-slideshow/cinnamon-slideshow.py")
--- a/files/usr/lib/cinnamon-menu-editor/cme/config.py
+++ b/files/usr/lib/cinnamon-menu-editor/cme/config.py
@@ -1,8 +1,8 @@
 prefix="/usr"
 datadir="/usr/share"
 localedir=datadir+"/locale"
-libdir="/usr/lib"
-libexecdir="/usr/lib/cinnamon-menu-editor"
+libdir="/usr/share/cinnamon"
+libexecdir="/usr/share/cinnamon/cinnamon-menu-editor"
 PACKAGE="cinnamon-menu-editor"
 VERSION="1.6.1"
 GETTEXT_PACKAGE="cinnamon"
--- a/files/usr/lib/cinnamon-settings/bin/Spices.py
+++ b/files/usr/lib/cinnamon-settings/bin/Spices.py
@@ -79,7 +79,7 @@
         
         self.window = window
         self.builder = Gtk.Builder()
-        self.builder.add_from_file("/usr/lib/cinnamon-settings/cinnamon-settings-spice-progress.ui")
+        self.builder.add_from_file("/usr/share/cinnamon/cinnamon-settings/cinnamon-settings-spice-progress.ui")
         self.progress_window = self.builder.get_object("progress_window")
         self.progress_window.set_transient_for(window)
         self.progress_window.set_destroy_with_parent(True)
